src/test/resources/gradle_plugin/pipeline_test_build_scan.txt

```diff
@@ -0,0 +1,12 @@
+stage ('build_Project') {
+    steps {
+        script {
+           if(isUnix()) {
```

Comment:
Sorry i didn't have time to take a look at this earlier.
Using gradle now as described in: https://jenkins.io/doc/book/pipeline/syntax/#tools
Thanks for the additional input!